PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx

```diff
@@ -258,18 +273,28 @@ struct FlowGFWOmegaXi {
   void FillProfilepT(const GFW::CorrConfig& corrconf, const ConstStr<chars...>& tarName, const int& ptbin, const int& partical, const float& cent)
   {
     int nMassBins = 0;
+    int nptbins = nPtBins;
     TAxis* fMass = nullptr;
+    TAxis* fpt = nullptr;
     if (partical == 3312) {
       nMassBins = nXiMassBins;
+      nptbins = nXiPtBins;
+      fpt = fXiPtAxis;
       fMass = fXiMass;
     } else if (partical == 3334) {
       nMassBins = nOmegaMassBins;
+      nptbins = nXiPtBins;
+      fpt = fXiPtAxis;
       fMass = fOmegaMass;
     } else if (partical == 310) {
       nMassBins = nK0sMassBins;
+      nptbins = nV0PtBins;
+      fpt = fV0PtAxis;
       fMass = fK0sMass;
     } else if (partical == 3122) {
```

Comment:
Please, use named PDG codes
You have them including
#include <TPDGCode.h>
and you can check the ones available in https://root.cern/doc/master/TPDGCode_8h.html
Extend it to other parts of the source code